optuna/visualization/matplotlib/_optimization_history.py

```diff
@@ -51,11 +56,17 @@ def objective(trial):
             study = optuna.create_study(sampler=sampler)
             study.optimize(objective, n_trials=10)
 
+            plt.tight_layout()
```

Comment:
I think we need to run this `plt.tight_layout()` after Optuna visualization. Please move this line after the next line.

Reply:
sorry for my mistake. I fixed it.